### PR TITLE
Update hp-es.babelon.tsv

### PR DESCRIPTION
**Pull request by MarioCazalla**
Update hp-es.babelon.tsv @matentzn

### CODE REVIEW

File: babelon/hp-es.babelon.tsv [hunk 1]
```diff
@@ -18047,4 +18047,916 @@ en	es	HP:5200426	Delusions of infestation	rdfs:label	Delirios de infestación	OF
...
+en	es	HP:6000701	Elevated urinary bile acid level	rdfs:label	Aumento del nivel de ácidos biliares en orina	OFFICIAL	The amount of bile acid in the urine, normalized for urine concentration, is above the upper limit of normal.
+en	es	HP:6000702	Elevated urinary glyoxylic acid level	rdfs:label	Aumento del nivel de ácido glioxílico en orina	OFFICIAL	The amount of glyoxylic acid in the urine, normalized for urine concentration, is above the upper limit of normal. The conjugate base of glyoxylic acid is known as glyoxylate.
+en	es	HP:6000703	Elevated circulating 3-hydroxyanthranilic acid concentration	rdfs:label	Aumento de la concentración de ácido 3-hidroxiantranílico en sangre	OFFICIAL	The concentration of 3-hydroxyanthranilic acid in the blood circulation is above the upper limit of normal.
+en	es	HP:6000704	Elevated circulating CXCL9 concentration	rdfs:label	Aumento de la concentración de CXCL9 en sangre	OFFICIAL	The concentration of chemokine (C-X-C motif) ligand 9 in the blood circulation is above the upper limit of normal.;inheres_in;inheres_in_part_of;part_of
```
Comment (matentzn) [inline babelon/hp-es.babelon.tsv:18962]:
```suggestion
en	es	HP:6000704	Elevated circulating CXCL9 concentration	rdfs:label	Aumento de la concentración de CXCL9 en sangre	OFFICIAL	The concentration of chemokine (C-X-C motif) ligand 9 in the blood circulation is above the upper limit of normal.
```

Seems like a copy/paste error?

**Review by matentzn — CHANGES_REQUESTED** [review]
Tiny suggestion @MarioCazalla! What do you think?

File: babelon/hp-es.babelon.tsv (hunk 1 above)
Comment (MarioCazalla) [reply babelon/hp-es.babelon.tsv:18962]: Yes, you are right. We have corrected this bug and also the 3 typos with mistakes

**Review by matentzn — APPROVED** [review]
Thank you, this looks great!

**Pull request merged**